index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+    />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="menu-bar">
+          <li id="menu-icon">Welcome</li>
+          <li>
+            <div class="menu-icon1"><i class="fas fa-bars"></i></div>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section>
+      <div class="container">
+        <div class="heading-container">
+          <h1 class="heading">Hey there. I’m Mariane.</h1>
+          <h2 class="heading-2">I'm a software developer</h2>
```

Comment:
Kindly add some left and right padding to the header titles and increase the font size to look more like the Figma design template.